### ISSUE TITLE
PR badge should point to our own contributing guide

### ISSUE THREAD

**Issue by StoneyJackson** [LibreFoodPantry/BEAR-Necessities-Market#90]
Of course, we need a contributing guide written.

**Comment by dtran2017** [comment]
Currently working on writing the contribution guide - are there specific points you need in the guide (such as licensing, system prerequisites, how to get started, etc.) ? So far I have:

- A brief description of what BEAR-Necessities-Market is
- A Code of Conduct
- Open Development (AKA how to find our pages on GitHub)
- Bugs (How to find known issues and reporting new issues)
- How to Contact Us (GitHub)
Note: All points are under development

**Comment by StoneyJackson** [comment]
Some of this work should be done at the LibreFoodPantry level, in the LibreFoodPantry/Community project. This project responsible for general organization and coordination across across LibreFoodPantry projects. Part of its responsibility is to define common guidelines and policies that are shared by all LibreFoodPantry projects.

There are a number of issues in LibreFoodPantry/Community that are related to this issue:

* https://github.com/LibreFoodPantry/Community/issues/29
* https://github.com/LibreFoodPantry/Community/issues/25
* https://github.com/LibreFoodPantry/Community/issues/24
* https://github.com/LibreFoodPantry/Community/issues/23
* https://github.com/LibreFoodPantry/Community/issues/6

Individual projects are likely to need project-specific documentation due to differences in technologies. However, whenever possible, we should try to define common documentation in the same place.

@dtran2017  We have been drafting these common documents in [a publicly visible google doc](https://docs.google.com/document/d/1me12Y9fP2-UglFHL6NAIKl6jpdzs97HVxPM97usFKDM/edit?usp=sharing) and then moving them to the README.md in LibreFoodPantry/Community . We should probably move to a pull-request contribution model. I'll start setting that up tomorrow. For now, just take a look at what we've been working on so far. Feedback is welcome.

**Comment by dtran2017** [comment]
At this point in time, is the focus a common contribution guide for the LibreFoodPantry/Community project versus a BEAR-Necessities-Market's version of a contribution guide? I was under the impression that this issue was specific for the BEAR-Necessities-Market project?

**Comment by StoneyJackson** [comment]
Yes, this issue is specific to BEAR-Necessities-Market. However, if it is possible to have a common contributing guide for all LibreFoodPantry projects, then we should do that.

I'm imagining that there would be one general contributing guide, and then developer specific documentation for each project. I believe a common contributing guide is possible because all the projects are using a common development platform (GitHub at the moment). The developer specific documentation would be necessary because the technologies for each project may be different.

Having a common contributing guide has several advantages:

- Saves time when launching new projects, as there is no need to develop a new guide from scratch.
- Developers that have worked on one LFP project would have a good idea how to work on another.
- The guide can be updated based on lessons learned from any one of the projects, and thus all projects can benefit from these lessons.

So, I would say that the focus should be on getting a common guide, and figure out how to deploy it, and incorporate it into the different projects.

BTW, I invited you as a contributor to LibreFoodPantry/Community so you can more easily help out there.

**Comment by StoneyJackson** [comment]
OK, I was mistaken... I invited you as a member of https://github.com/LibreFoodPantry . If that doesn't give you the permissions you need, let me know.
